scripts/ai-sdlc/generate-spec.mjs

```
@@ -6,42 +6,122 @@
+  let currentLevel = [rootDir];
+  while (currentLevel.length > 0 && results.length < BUDGET_PER_ROOT * 5) {
+    const nextLevel = [];
+    for (const dir of currentLevel) {
+      let entries;
+      try {
+        entries = readdirSync(dir);
+      } catch {
+        continue;
+      }
+      for (const name of entries) {
+        if (name.startsWith('.') || name === 'node_modules' || name === 'dist') continue;
```

ESLint

[error] Expected { after 'if' condition.

(curly)